src/index.js

```diff
@@ -27,16 +26,29 @@ addIcon.className = 'addicon';
 addButton.appendChild(addIcon);
 addListDiv.appendChild(inputTask);
 addListDiv.appendChild(addButton);
-
 let tasks = JSON.parse(localStorage.getItem('todolist')) || [];
+function updateTaskListItemClass(taskId, completed) {
+  const taskListItem = document.querySelector(`[data-task-id="${taskId}"]`);
+  if (taskListItem) {
+    if (completed) {
+      taskListItem.classList.add('checked');
+      taskListItem.style.background = '#f4f5Cf';
+      taskListItem.style.opacity = '0.5';
+    } else {
+      taskListItem.classList.remove('checked');
+      taskListItem.style.background = 'none';
+      taskListItem.style.opacity = '1';
```

Comment:
- [ OPTIONAL ] Great job working on the functionality for adding and removing but I think it would be great if you fix the indexes since when I try to remove the second task the indexes are not updated to be 1, 2, 3 it becomes 1, 3, 4 but it should be ordered as mentioned in the requirements so please fix that ✨🤩I am assigning as optional because of this is part from the previous milestone but it still very important to implement 🤩

![image](https://github.com/Nessrine88/Todolist/assets/58553711/dade07c2-f804-4630-b69a-9ecc5783fa1b)

Reply:
Thank you so much for the feedback. I will try to implement it soon.